doc/GettingStarted.md

```diff
@@ -34,10 +34,7 @@ The `dotnet` templates included in the `Uno.Templates` package are used to easil
 The generated solution will contain:
 
 * *MyProjectName* - for application logic, and other constructs like view models and services, as well as the pages, controls and other views that make up the UI of the application
-* *MyProjectName.DataContracts* - for entities that are shared with an API backend.
-* *MyProjectName.Server* - ASP.NET project that hosts a WebAPI and can be used to host the WASM application.
```

Comment:
Note that the server project is still present. It cannot be merged with the main project.